source/development_process/index.rst

```diff
@@ -0,0 +1,41 @@
+The OSL Development Workflow
+============================
+
+The OSL builds websites for a variety of clients including the Oregon Virtual
+School Districts, the Oregon Sea Grant Foundation, the OSU Biology department
+and Google. When the initial development is finished, the OSL will host and
+maintain the websites. This document will guide developers through the steps of
+creating a new website for a client. New developers are encouraged to read this
+document to understand how the developer team works, and experienced developers
+are encouraged to review this document to brush up their knowledge of parts of
+the process they may not be intimately familiar with. This workflow description
+will also be available to clients so they can understand how the OSL developer
+team works.
+
+.. toctree::
+  :maxdepth: 1
+  :glob:
+
+  *
+
+Tools The Team Uses
+-------------------
+
+
+At the OSL developers use a variety of open source tools. Developers are
+expected to have at least passing familiarity with all of these tools. New
+developers are encouraged to research any of these tools they have not heard
+of.
```

Comment:
Prescriptivist! The only reason people believe this is incorrect is because you can't do this in Latin!

/me teases